modules/psap-driver-toolkit-pulling.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/psap-driver-toolkit.adoc
+
+[id="pulling-the-driver-toolkit"]
+= Pulling the Driver Toolkit container image
+
+The driver-toolkit image is available on registry.redhat.io and in the OpenShift release payload. The image corresponding to the most recent minor release of {product-title} will be tagged with the version number on registry.redhat.io. The image URL for a specific release can be found using the `oc adm` CLI command.
+
+[id="pulling-the-driver-toolkit-from-registry"]
+== Pulling the Driver Toolkit container image from registry.redhat.io
```

Comment:
There are instructions at - https://catalog.redhat.com/software/containers/openshift4/driver-toolkit-rhel8/604009d6122bd89307e00865?tag=v4.7.0-202104142050.p0&push_date=1619452122000&container-tabs=gti - on how to pull this image. Should we just refer to that ? Which one is the source of truth ?

Reply:
Thanks for pointing that out, I think these instructions are sufficient and we should refer to them.